test/state/bloom.hpp

```diff
@@ -0,0 +1,43 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include "hash_utils.hpp"
+#include <intx/intx.hpp>
+#include <span>
+
+namespace evmone::state
+{
+struct TransactionReceipt;
+struct Log;
+
+/// The 2048-bit hash suitable for keeping an Ethereum bloom filter value.
+struct BloomFilter
+{
+    /** The 256 bytes of the bloom filter value. */
```

Comment:
```suggestion
    //// The 256 bytes of the bloom filter value.
```

Reply:
@chfast Do we have any document which defines this details?

Reply:
No. We just use single-line C/C++ comments.